tests/install/boot.pm

```diff
@@ -1,8 +1,14 @@
 use Mojo::Base 'openQAcoretest';
+use testapi;
 use utils;
 
+use OpenQA::Wheel::Launcher 'start_gui_program';
+
+sub _prevent_inactivity_timeout { start_gui_program('xset s off -dpms', valid => 1) }
+
 sub run {
     wait_for_desktop;
+    _prevent_inactivity_timeout;
```

Comment:
I would prefer that the boot module really only does what it says and stays that simple. How about a separate test module where you execute that?